requirements/environment.yml

```diff
@@ -9,34 +9,67 @@ dependencies:
   - annotated-types=0.7.0
   - astroid=3.3.5
   - asttokens=2.4.1
-  - black=24.8.0
+  - attrs=24.2.0
+  - aws-c-auth=0.7.31
```

Comment:
I guess these requirements were updated automatically, but why do we need AWS and Azure stuff for reading GRIB files?

Reply:
Well, yes that happens by `./setup_env.sh -u -e` when we want to update the pinned environment. So I don't know why theses are added for earthkit.data. Maybe there is some backend in there to use aws or azure. But I don't know.

Reply:
Ok, no worries. I was mainly checking just in case you knew we absolutely don't need those.